Terminal Commands.md

```diff
@@ -48,17 +67,56 @@ Some commands can be found in the `bin` directory.
 `>` - used to store the output of a task in some file(overwrites if file with same name is present) rather than displaying it on the terminal.<br/>
 `>>` - same task as `>` but does not overwrites just appends to the file with the same name.<br/>
 
-`cat` - used to open a file in terminal in read-only format
-
 `|` - piping is used to give the output of a command as input to another command ..for e.g `history | grep "find"` will search for "find" in the output of `history`
 
 `<command> | tee <filename>` - used to show output of `command` on terminal as well as writing the output to a file `filename`.<br/>
 
+`cat` - used to open a file in terminal in read-only format<br>
+`cat file1 > file2` - copies content of file1 one to file2<br>
```

Comment:
This is not required since we have already mentioned the use of `>`. You can keep the scrolling part as a Trivia though